run_tests.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# Check if in the right working directory.
+if [ ! -d "webapp" ]; then
+  echo "You are supposed to run this script in the Deadline root directory."
+  exit 1
+fi
+
+# Check if in Deadline virtual environment.
+source venv/deadline/bin/activate || exit 1;
```

Comment:
same question as in the other PR—does this have to match the readme?

Reply:
fixed readme!